### PR TITLE
Fix splitting of list

### PR DESCRIPTION
**Pull request by BonaBeavis**
Fixes #

### Proposed fixes:

Fixes: https://github.com/ckan/ckan/issues/7525

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

File: ckan/cli/views.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def create(ctx: click.Context, types: list[str], dataset: list[str],
     """
 
     datastore_enabled = (
-        u"datastore" in config[u"ckan.plugins"].split()
+        u"datastore" in config[u"ckan.plugins"]
```
[the remark this answers is not in the document]
Comment (BonaBeavis) [reply ckan/cli/views.py]: I added the requested change. I did it with the GitHub GUI, so the commit history sucks, can I/you squash the commit before merging?

**Comment by BonaBeavis** [comment]
Ping?

**Comment by amercader** [comment]
Thanks @BonaBeavis, sorry it took a while

**Comment by amercader** [comment]
Merged by mistake to `2.10`. I cherry-picked it to master and dev-v2.10

**Pull request merged**